test/segmentation/test_stitching.py

```diff
@@ -1,15 +1,48 @@
 import unittest
 
-from elf.evaluation import rand_index
+import numpy as np
 from skimage.data import binary_blobs
 from skimage.measure import label
 
+from elf.evaluation import rand_index
+
 
 class TestStitching(unittest.TestCase):
     def get_data(self, size=1024, ndim=2):
         data = binary_blobs(size, blob_size_fraction=0.1, volume_fraction=0.2, n_dim=ndim)
         return data
 
+    def get_tiled_data(self, size=1024, ndim=2, tile_shape=(512, 512)):
+        data = self.get_data(size=size, ndim=ndim)
+        data = label(data)  # Ensure all inputs are instances (the blobs are semantic labels)
+
+        # Create tiles out of the data.
+        # Ensure offset for objects per tile to get individual ids per object per tile.
+        import nifty.tools as nt
+        blocking = nt.blocking([0] * ndim, data.shape, tile_shape)
+        n_blocks = blocking.numberOfBlocks
+
+        offset = 0
+        bb_tiles, tiles = [], []
+        for tile_id in range(n_blocks):
```

Comment:
It would be a bit more elegant to just modify the labels in place here.

Reply:
Yes, true. I'll take care of it. Thanks!